README.md

```diff
@@ -1,5 +1,4 @@
 # Medallia Conversations Adapter
-
```

Comment:
Leave this in for formatting sync with other projects.  (Style guidelines say to have a blank line after each Markdown heading.)